src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -456,6 +465,20 @@ def engine_forward(
                         and not self.force_max_tokens
                     ):
                         break
+                    if stop is not None:
+                        decoded_token = self.tokenizer.decode(token)
+
+                        decoded_token = (
+                            decoded_token
+                            if decoded_token.isspace()
+                            else decoded_token.strip()
+                        )
+                        if decoded_token in stop:
+                            _LOGGER.info(
```

Comment:
Should be debug log

Reply:
Updated, was doing it nonetheless when I saw your comment on other PR